### PR TITLE
OCP4/PCI-DSS: Add responses to Requirement 5

### PR DESCRIPTION
**Pull request by JAORMX**
These mostly pertain to threat detection and malware. Our supported
solution is the File Integrity Operator. However, vulnerability scans
via image registries are also documeted, even if they're out of scope of
our automation.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

File: controls/pcidss_ocp4.yml [hunk 1]
```diff
@@ -925,28 +925,48 @@ controls:
...
+    OpenShift that allow for this.
+  status: automated
+  controls:
+    - id: Req-5.1.1
+      title: 5.1.1 Ensure that anti-virus programs are capable of detecting, removing,
+        and protecting against all known types of malicious software.
+      levels:
+      - base
+      notes: |-
+        OpenShift container platforms may install the  the OpenShift File
```
Comment (jhrozek) [inline controls/pcidss_ocp4.yml:943]: nit: the the

File: controls/pcidss_ocp4.yml [hunk 2]
```diff
@@ -956,9 +976,20 @@ controls:
     * Generate audit logs which are retained per PCI DSS Requirement 10.7.
   levels:
   - base
-  notes: ''
-  status: pending
-  rules: []
+  notes: |-
+    OpenShift container platforms may install the  the OpenShift File
```
Comment (jhrozek) [inline controls/pcidss_ocp4.yml:980]: the the

File: controls/pcidss_ocp4.yml [hunk 3]
```diff
@@ -956,9 +976,20 @@ controls:
...
-  notes: ''
-  status: pending
-  rules: []
+  notes: |-
+    OpenShift container platforms may install the  the OpenShift File 
+    Integrity Operator [1] which monitors file system integrity on the host.
+    This may allow for the detection of threats on the hosts which attempt
+    to modify the file system in malicious ways. Additionally, there exist
+    several solutions to scan for container vulnerabilities which are indispensible
+    from any deployment. One such example is Red Hat Quay [2] which supports for
```
Comment (jhrozek) [inline controls/pcidss_ocp4.yml:985]: from or for?

File: controls/pcidss_ocp4.yml [hunk 4]
```diff
@@ -925,28 +925,48 @@ controls:
...
+        and protecting against all known types of malicious software.
+      levels:
+      - base
+      notes: |-
+        OpenShift container platforms may install the  the OpenShift File 
+        Integrity Operator [1] which monitors file system integrity on the host.
+        This may allow for the detection of threats on the hosts which attempt
+        to modify the file system in malicious ways. Additionally, there exist
+        several solutions to scan for container vulnerabilities which are indispensible
+        from any deployment. One such example is Red Hat Quay [2] which supports for
```
Comment (jhrozek) [inline controls/pcidss_ocp4.yml:948]: from or for?

**Review by jhrozek — COMMENTED** [review]
the content itself is good, but see inline for some nits

**Comment by Vincent056** [comment]
I think it looks good besides what @jhrozek mentioned!

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 7f059a4be0b28dfa2a48a224ccd5c4fdf94dce02

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**